packages/ag-charts-community/src/chart/tooltip/tooltip.ts

```diff
@@ -64,13 +64,13 @@ export interface TooltipContent {
     title?: string;
     symbol?: LegendSymbolOptions;
     data?: TooltipContentDataRow[];
+    rawHtmlString?: string;
 }
```

Comment:
This is one or the other. Might be better to represent like this:-

```suggestion
    title?: string;
    symbol?: LegendSymbolOptions;
    data?: TooltipContentDataRow[];
} | {
    rawHtmlString?: string;
}
```

Reply:
I've tried applying this - but the simplest thing to do is add a `type` discriminator field, as `'rawHtmlString' in content` isn't enough of a hint to the TS compiler that on the negative control flow that the inferred typing of `content` is a single union member type.

https://github.com/ag-grid/ag-charts/pull/3359/commits/9e3aeac5c35bf6dcd1983b47ba308eaefbbc281f is the smallest additional change encoding use of a type union for `TooltipContent`.